packages/cli/src/commands/init.ts

```diff
@@ -432,11 +430,18 @@ async function createTriggerAppRoute(
   projectPath: string,
   path: string,
   options: ResolvedOptions,
+  isTypescriptProject: boolean,
   usesSrcDir = false
 ) {
-  const tsConfigPath = pathModule.join(projectPath, "tsconfig.json");
+  const configFileName = isTypescriptProject ? "tsconfig.json" : "jsconfig.json"
```

Comment:
Since it will not all be ts projects we needed to update the config file path.
I'm relying on the assumption here that all js projects will have a jsconfig.json file